end-to-end-tests/cypress/e2e/OSCAL/OSCAL_Load_Tests.cy.js

```diff
@@ -37,12 +37,26 @@ describe('Test Loading Component Definitions', () => {
   })
 })
 
-describe('Test Loading Wrong Object Type', () => {
-  it('Displays Proper Error on Load of Wrong Object Type', () => {
-    cy.visit(Cypress.env('base_url')) 
-    cy.contains("REST Mode").click()
-    cy.contains('OSCAL Catalog URL').first().should('exist').next().click().clear().type("https://raw.githubusercontent.com/usnistgov/oscal-content/main/examples/ssp/json/ssp-example.json")
-    cy.contains('Reload').click()
+describe('Errors caused by loading a bad component definition', () => {
+  beforeEach(() => {
+    cy.navToCdefEditor(COMP_DEF_TITLE_ORIG);
+    cy.contains("REST Mode").click();
+    cy.contains('OSCAL Component URL').first().should('exist').next().click().clear().type("https://raw.githubusercontent.com/usnistgov/oscal-content/main/examples/ssp/json/ssp-example.json");
+    cy.contains('Reload').click();
+  });
+
+  it('display "yikes" on load of wrong object type', () => {
     cy.contains('Yikes').should('be.visible')
-  })
-})
\ No newline at end of file
+  });
+
+  it('do not persist after loading a valid component in Viewer', () => {
```

Comment:
Maybe I am misunderstanding something, but why would there be an issue of persistence here or after loading a valid component?

Reply:
You are right that there shouldn't be. https://github.com/EasyDynamics/oscal-react-library/pull/490 fixed it and this test is to make sure we do not reintroduce that error. 

Essentially if I loaded a bad `component` in the viewer I would get the standard error message. Typing in the url of a valid `Component` and then hitting `Reload` should make that error go away. It was not doing that.